frontend/sw360-portlet/src/main/webapp/WEB-INF/liferay-portlet.xml

```diff
@@ -319,6 +319,17 @@
         <header-portlet-javascript>/webjars/jquery/1.12.4/jquery.min.js</header-portlet-javascript>
     </portlet>
 
+    <portlet>
+        <portlet-name>WsImport</portlet-name>
+        <icon>/icon.png</icon>
+        <use-default-template>false</use-default-template>
+        <instanceable>false</instanceable>
+        <header-portlet-css>/css/main.css</header-portlet-css>
+        <header-portlet-javascript>/js/main.js</header-portlet-javascript>
+        <header-portlet-javascript>/webjars/jquery/1.12.4/jquery.min.js</header-portlet-javascript>
+        <add-default-resource>true</add-default-resource>
```

Comment:
From [https://docs.liferay.com/portal/4.2/official/liferay-portlet-development-guide-4.2/multipage/ch02.html](liferay-portlet-development-guide-4.2)
> If the add-default-resource value is set to true, the default portlet resources and permissions are added to the page.

Why is this necessary? It was added in https://github.com/eclipse/sw360/pull/270/commits/b6641a6a2606d3a043d2576878e47afffc08ac76 but does not look related to "feat(wsimport): no portlet borders".

Reply:
It was unnecessary line of code and is now removed. It was copied from a snipped of code that removed portlet's borders by default.